packages/validation-helpers/src/api/schemas/scanner_schemas.ts

```diff
@@ -71,7 +71,7 @@ export const PostPackageConfigurationRes = z.object({
     licenseConclusions: z.array(
         z.object({
             path: z.string(),
-            detectedLicenseExpressionSPDX: z.string(),
+            detectedLicenseExpressionSPDX: z.nullable(z.string()),
```

Comment:
After this change, does backend now return null instead of "" when no detected license? I need to sync the change with ORT.

Reply:
Yes, although this now reminds me that it's still possible to add a license conclusion with an empty string as detectedLicenseExpressionSPDX, which isn't currently transformed to null before saving to database. Perhaps I should fix that before merging to main.

Reply:
Or it could be left the way it is, until you have implemented the changes on ORT side

Reply:
I believe it would be best to leave as it is now.

Reply:
Yes, it will return null